mslice/plotting/pyplot.py

```diff
@@ -3041,26 +3068,27 @@ def triplot(*args, **kwargs):
 
 # This function was autogenerated by boilerplate.py.  Do not edit as
 # changes will be lost
-@_autogen_docstring(Axes.violinplot)
-@script_log(module_name)
-def violinplot(dataset, positions=None, vert=True, widths=0.5, showmeans=False,
-               showextrema=True, showmedians=False, points=100, bw_method=None,
-               hold=None, data=None):
-    ax = gca()
-    # allow callers to override the hold state by passing hold=True|False
-    washold = ax.ishold()
-
-    if hold is not None:
-        ax.hold(hold)
-    try:
-        ret = ax.violinplot(dataset, positions=positions, vert=vert,
-                            widths=widths, showmeans=showmeans,
-                            showextrema=showextrema, showmedians=showmedians,
-                            points=points, bw_method=bw_method, data=data)
-    finally:
-        ax.hold(washold)
-
-    return ret
+if not MPL_COMPAT:
+    @_autogen_docstring(Axes.violinplot)
+    @script_log(module_name)
+    def violinplot(dataset, positions=None, vert=True, widths=0.5, showmeans=False,
```

Comment:
Yes. I think so, it gives an `IndentationError` otherwise. 

Actually, `Axes.violinplot`, `Axes.angle_spectrum`, `Axes.phase_spectrum` and `Axes.magnitude_spectrum` were all introduced in matplotlib 1.4, so don't strictly need matplotlib 1.5 - but I'm testing it on a system with matplotlib 1.3.1. Should we be a bit more fine grained in the check?